<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QDialog" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>564</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>564</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mapuche</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>563</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>13</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="newMapBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>New Map</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openMapBtn">
        <property name="text">
         <string>Open Map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newTmBtn">
        <property name="text">
         <string>New Tilemap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadTmBtn">
        <property name="text">
         <string>Load Tilemap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mapPropBtn">
        <property name="text">
         <string>Map Properties</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QRadioButton" name="layer0Radio">
        <property name="text">
         <string>Layer 0</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="layer1Radio">
        <property name="text">
         <string>Layer 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="layer2Radio">
        <property name="text">
         <string>Layer 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="layer3Radio">
        <property name="text">
         <string>Layer 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="layer4Radio">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Layer 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="layerEventRadio">
        <property name="text">
         <string>Event Layer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="fillBtn">
        <property name="text">
         <string>Fill</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="defEventBtn">
        <property name="text">
         <string>Define Event</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>openMap()</slot>
 </slots>
</ui>
